plugins/content/fields/src/Extension/Fields.php

```diff
@@ -68,7 +68,7 @@ public function onContentPrepare($context, &$item, &$params, $page = 0)
         }
 
         // Prepare the full text
-        if (property_exists($item, 'fulltext') && strpos($item->fulltext, 'field') !== false) {
+        if (property_exists($item, 'fulltext') && is_string($item->fulltext) && strpos($item->fulltext, 'field') !== false) {
```

Comment:
What about the following code which makes it a bit easier to read?
```suggestion
        if (!empty($item->fulltext) && strpos($item->fulltext, 'field') !== false) {
```